frontend/hub/collections/hooks/useCopyToRepository.tsx

```diff
@@ -233,7 +236,7 @@ export async function copyToRepositoryAction(
   }
 
   const pipeline = collection.repository?.pulp_labels?.pipeline;
-  if (!(pipeline === 'staging' || pipeline === 'rejected')) {
+  if (operation === 'approve' && !(pipeline === 'staging' || pipeline === 'rejected')) {
```

Comment:
This caused the problems, the approvals have this condition to allow only approvals from staging or rejected, but it was failing in copy (which is allowed in UI only from approved repos).